specs/Proposals/0040-split-states.md

```diff
@@ -206,52 +271,46 @@ fn apply_chunks(...) -> Result<(), Error> {
             );
         }
     }
-    ... 
+    ...
+
 }
 ```
 
 ## Garbage collection
 #TODO
 ???
+I think garbage collection should mostly work automatically? Need to verify that new ChunkExtra stored for these new shards will be garbage collected correctly.
+Otherwise I think it should be fine.
```

Comment:
We need to handle garbage collection of the old state after it is split into multiple. Today garbage collection won't automatically clean that. This can be done later though